src/icons.rs

```diff
@@ -5,280 +5,356 @@ use lazy_static::lazy_static;
-        "update" => " \u{e8d7} ",
-        "uptime" => " \u{e192} ", // Same as time symbol.
-        "volume_empty" => " \u{e04e} ",
-        "volume_full" => " \u{e050} ",
-        "volume_half" => " \u{e04d} ",
-        "volume_muted" => " \u{e04e} \u{e04f} ",
-        "xrandr" => " \u{e31e} "
+        "bat_charging" => "\u{e1a3} ",
+        "bat_discharging" => "\u{e19c} ",
+        "bat_empty" => "\u{e19c} ",
+        "bat_full" => "\u{e1a4} ",
+        "bat_half" => "\u{e1a5} ",
+        "bat_quarter" => "\u{e1a5} ",
+        "bat_three_quarters" => "\u{e1a5} ",
+        "bell" => "\u{e7f4} ",
+        "bell-slash" => "\u{e7f8} ",
+        "bluetooth" => "\u{e1a7}",
+        "calendar" => "\u{e935}",
+        "cogs" => "\u{e8b8} ",
+        "cpu" => "\u{e640} ",
+        "disk_drive" => "\u{e1db} ",
+        "docker" => "\u{e532} ",
+        "github" => "\u{e86f} ",
+        "gpu" => "\u{e333} ",
+        "headphones" => "\u{e60f}",
+        "joystick" => "\u{e30f}",
+        "keyboard" => "\u{e312}",
+        "mail" => "\u{e0be} ",
+        "memory_mem" => "\u{e322} ",
+        "memory_swap" => "\u{e8d4} ",
+        "mouse" => "\u{e323}",
+        "music" => "\u{e405} ",
+        "music_next" => "\u{e044} ",
+        "music_pause" => "\u{e034} ",
+        "music_play" => "\u{e037} ",
+        "music_prev" => "\u{e045} ",
+        "net_loopback" => "LO",
+        "notification" => "\u{e7f7}",
+        "phone" => "\u{e324} ",
+        "phone_disconnected" => "\u{1f4f5}",
+        "pomodoro" => "\u{1f345}",
+        "resolution" => "\u{f152}", // crop-square-rounded
+        "tasks" => "\u{e8f9}",
+        "thermometer" => "\u{f2c8}", // TODO
+        "time" => "\u{e192}",
+        "toggle_off" => "\u{e836}",
+        "toggle_on" => "\u{e837}",
+        "update" => "\u{e8d7}",
+        "uptime" => "\u{e192}", // Same as time symbol.
+        "volume_empty" => "\u{e04e}",
+        "volume_full" => "\u{e050}",
+        "volume_half" => "\u{e04d}",
+        "volume_muted" => "\u{e04e} \u{e04f}",
+        "xrandr" => "\u{e31e}"
+    };
+
+    // Material an in NerdFont
+    pub static ref MATERIAL_NF: Map<String, String> = map_to_owned! {
+        "" => "",
+        "backlight_empty" => "\u{e38d}",
+        "backlight_full" => "\u{e39b}",
+        "backlight_partial1" => "\u{e3d1}",
+        "backlight_partial2" => "\u{e3ce}",
+        "backlight_partial3" => "\u{e3cb}",
+        "bat_charging" => "\u{f583}",
+        "bat_discharging" => "\u{f57d}",
+        "bat_empty" => "\u{f58d}",
+        "bat_full" => "\u{f578}",
+        "bat_half" => "\u{f57d}",
+        "bat_quarter" => "\u{f57a}",
+        "bat_three_quarters" => "\u{f57f}",
+        "bell" => "\u{f599}",
+        "bell-slash" => "\u{f59a}",
+        "bluetooth" => "\u{f5ae}",
+        "calendar" => "\u{f5ec}",
+        "cogs" => "\u{f992}",
+        "cpu" => "\u{f9c4}",
+        "disk_drive" => "\u{f7c9}",
+        "docker" => "\u{f308}", // nf-linux-docker
```

Comment:
Can you add these comments for the other icons as well? It makes it much easier to figure out the intention.

Can get them from here: https://www.nerdfonts.com/cheat-sheet

Reply:
Will take some time.